app/views/tokens/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= search_form_for @q do |f| %>
```

Comment:
You should update this form to use Turbo for searching. You can use `people/index.html.erb` for reference.